test/query_test.js

```diff
@@ -1243,6 +1243,18 @@ describe('query', function () {
     return Promise.all([p1, p2]);
   });
 
+  it('to_double', function () {
+    var p1 = assertQuery(query.ToDouble(3.14), 3.14);
+    var p2 = assertQuery(query.ToDouble("3.14"), 3.14);
```

Comment:
actually you are just testing if a double is converted to a double, add another test converting a "integer" to double.

```javascript
assertQuery(query.ToString(query.ToDouble(3.14)), "3.14");
assertQuery(query.ToString(query.ToDouble(10)), "10.0");
```